src/festivals/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from versatileimagefield.fields import VersatileImageField
+from ckeditor_uploader.fields import RichTextUploadingField
+
+
+class Festival(models.Model):
+    FEST_CHOICES = (
+        ('IGNS', 'Ignus'),
+        ('VRCHS', 'Varchas'),
+        ('SPNDN', 'Spandan'),
+        ('NMBL', 'Nimble'),
+    )
+    name = models.CharField(max_length=5, choices=FEST_CHOICES)
+    photo = VersatileImageField(upload_to='festival')
+    about = models.TextField()
+    link = models.URLField(blank=True, null=True, default=None)
+    published = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.get_name_display()
+
+
+class FestivalEventCategory(models.Model):
+    name = models.CharField(max_length=128)
+    festival = models.ForeignKey(Festival, on_delete=models.CASCADE)
+    cover = VersatileImageField(upload_to='festival_event_category', blank=True)
+    slug = models.SlugField()
+    about = RichTextUploadingField()
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        ordering = ['name']
+        verbose_name = 'Festival Event Category'
+        verbose_name_plural = 'Festival Event Categories'
+
+
+class FestivalEvent(models.Model):
```

Comment:
Change `FestivalEvent` to `Event`

Reply:
Won't this cause issues later with the normal events for the gymkhana clubs
Someone might get confused

Similarly for FestivalEventCategory

Reply:
These will fall under festivals app, you've provided the verbose names in Meta class though, however, class name should be changed as suggested because it is responsible for internal table structure.